audit/organizations/kubernetes.io/roles/audit.viewer.json

```diff
@@ -149,6 +149,8 @@
     "bigquery.reservations.list",
     "bigquery.routines.get",
     "bigquery.routines.list",
+    "bigquery.rowAccessPolicies.getIamPolicy",
+    "bigquery.rowAccessPolicies.list",
```

Comment:
Expected, this is from https://github.com/kubernetes/k8s.io/pull/2156 which picked up a refresh of aggregated permissions for audit.viewer